help/embeddings.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Protein Embeddings
+type: help
+categories: UniProtKB,Automatic_annotation,help
+---
+
+Protein embeddings is a way to encode functional and structural properties of a protein, mostly from its 
+sequence only, in a machine-friendly format (vector representation). Embeddings are computationally expensive, 
+however once computed they can be leveraged for different tasks such as sequence similarity search, sequence 
+clustering, and sequence classification.
+
+UniProt is providing raw embeddings (per-protein and per-residue using ProtT5 model) for UniProtKB/Swiss-Prot and some 
+reference proteomes of model organisms (such as Homo sapiens, Sars-CoV-2, and E.coli) with every release.
+
+**Note:** Proteins longer than 12k residues are excluded due to limitation of GPU memory (only a handful of proteins 
+are excluded). 
+
+## Sample code
+The embeddings h5 files store the embeddings as key-value-pairs. The key is the protein accession and the value is 
+the embeddings vector. The following code snippet shows how to read and iterate over an embeddings file in python.
+
+```python
+import numpy as np
+import h5py
+
+with h5py.File("path/to/embeddings.h5", "r") as file:
+    print(f"number of entries: {len(file.items())}")
+    for sequence_id, embedding in file.items():
+        print(
+            f"  id: {sequence_id}, "
+            f"  embeddings shape: {embedding.shape}, "
+            f"  embeddings mean: {np.array(embedding).mean()}"
+        )
+```
+
+###### per-protein file sample output:
```

Comment:
I would specify something like "as per release 2022_04" unless you're planning on updating this page ahead of each release. Otherwise if people load it in the future and don't get the same values they might think something is wrong.
Same thing for the other output

Reply:
No we don't plan to update the page with every release. Actually the sample output is not intended to be exact output from files but just to show the structure of data. Smallest dataset have 17 entries (SARS-CoV-2) and we don't want to show full output, so if we want to be more specific about the output maybe we can say something like:

###### per-protein file sample output (SARS-CoV-2 as per 2022_04 release):
```
number of entries: 17
  id: A0A663DJA2,   embeddings shape: (1024,),   embeddings mean: 0.0006136894226074219
  id: P0DTC1,   embeddings shape: (1024,),   embeddings mean: 0.0011968612670898438
  id: P0DTC2,   embeddings shape: (1024,),   embeddings mean: 0.001041412353515625
  ...
```

show the actual total entries in the file and use '...' to indicate more lines not shown. Same for the other output.
what do you think?

just a note that embeddings shape and vector isn't expect to change if the sequence doesn't change, but the proteome may change it number of proteins I guess.

Reply:
Okay, what you're suggesting for the slightly more specific output sounds good 👍🏽

Reply:
okay will make those changes.. thanks

Reply:
I have put the dataset and release info as header instead of repeating it.

Reply:
There are changes in requirements regarding putting the per-protein files and per-residue files in different ftp locations. Therefore, the download links need to be updated and also maybe the help page.
Will make those changes once we have the files copied over hopefully today.